### PR TITLE
Add 7.16.0-rc.9 and 6.16.0-rc.9

### PR DESCRIPTION
**Pull request by ogaca-dd**
### What does this PR do?

Add 7.16.0-rc.9 and 6.16.0-rc.9

rc9 contains only https://github.com/DataDog/datadog-agent/pull/4611 (not yet merged).
This PR will be merged once https://github.com/DataDog/datadog-agent/pull/4611 will be merged

### CODE REVIEW

**Review by remeh — APPROVED** [review]
Sanity check: you're still on the old tags for integrations and omnibus-software.

**Comment by ogaca-dd** [comment]
rc9 contains only https://github.com/DataDog/datadog-agent/pull/4611 (not yet merged).

File: release.json [hunk 1]
```diff
@@ -13,6 +13,20 @@
 		"JMXFETCH_VERSION": "0.33.1",
 		"JMXFETCH_HASH": "140dd8568b78f5a616da8664f34bb37397c3d3ed87af106d632714f7218e8e88"
 	},
+	"7.16.0-rc.9": {
+		"INTEGRATIONS_CORE_VERSION": "7.16.0-rc.8",
+		"OMNIBUS_SOFTWARE_VERSION": "7.16.0-rc.6",
```
Comment (derekwbrown) [inline release.json]: Is this right?  `rc6`?

File: release.json [hunk 2]
```diff
@@ -13,6 +13,20 @@
...
+		"OMNIBUS_RUBY_VERSION": "datadog-5.5.0",
+		"JMXFETCH_VERSION": "0.33.1",
+		"JMXFETCH_HASH": "140dd8568b78f5a616da8664f34bb37397c3d3ed87af106d632714f7218e8e88"
+	},
+	"6.16.0-rc.9": {
+		"INTEGRATIONS_CORE_VERSION": "7.16.0-rc.8",
+		"OMNIBUS_SOFTWARE_VERSION": "7.16.0-rc.6",
+		"OMNIBUS_RUBY_VERSION": "datadog-5.5.0",
```
Comment (derekwbrown) [inline release.json]: same.

**Review by derekwbrown — APPROVED** [review]
Approving in case the comments (rc6/rc8) are expected, we just didn't move the tag on omnibus-software.  

Otherwise, fix and go...

**Review by truthbk — APPROVED** [review]
If the `OMNIBUS_SOFTWARE` and `INTEGRATIONS_CORE_VERSION` tag discrepancies (RC6 and RC8 respectively) are good, then this looks good.

File: release.json (hunk 1 above)
Comment (ogaca-dd) [reply release.json]: Yes it is the same tag.

**Pull request merged**